Moving files to new repo.

MikeJLeon opened the pull request:
I'm simply moving all the files that were created on the old repo. Let me know if this is the right way.

MikeJLeon committed:
Moving files to new repo.

MikeJLeon committed:
Fixed Import issue

rberry206 reviewed (approved):
If it worked before it will work again.

MikeJLeon committed:
Changed fb name apply import fix

MikeJLeon committed:
Dao's Latest Code, Updated output file name, other changes

Added a dummy access code. Updated output to produce a json.
Matched output to Dao's output naming convention.
Changed terminal output, removed ignored link message and instead just ignore it before it is even added to the queue

MikeJLeon committed:
Remove a location requirement

Removed seattle requirement,  should be looking for all WA events now.

jams2018 reviewed (approved):
It was working in the old repo. It should work in the new. It needs to be approved.

daonguyen81 commented:
Looking good Mike, Thanks!
